Readme.md

```diff
@@ -86,20 +86,20 @@ chmod +x run-dev.sh
 ./setup-dev.sh
 ```
 
-### Run
+#### Run
 ```shell
 ./run-dev.sh
 # press CTRL+C in the terminal window to stop
 ```
-
-On a Mac http://localhost:8080 will open in your browser automatically. On some linux distros you may need to do it manually.
-
-For Debian 12 instructions, see [this post](https://github.com/builder555/PineSAM/discussions/47#discussion-4884758).
+* On a Mac http://localhost:8080 will open in your browser automatically.
+* On some linux distros you may need to manually open http://localhost:8080
+* For Debian 12 hints, see [this post](https://github.com/builder555/PineSAM/discussions/47#discussion-4884758).
+* If you run into problems, see [Known Issues](https://github.com/builder555/PineSAM/Readme.md#known-issues) below or the [Discussions](https://github.com/builder555/PineSAM/discussions) 
```

Comment:
probably more appropriate to put the link in the Known issues, and remove this line since it's an entire section on its own already

Reply:
moved last line to known issues